src/pyclass.rs

```diff
@@ -593,6 +595,7 @@ pub mod boolean_struct {
 }
 
 /// A trait which is used to describe whether a `#[pyclass]` is frozen.
+#[doc(hidden)]
```

Comment:
I've hidden this because it just seems...weird to be browsing the documentation and finding this. Imo this just just an implementation detail.